examples/ITK_Example18_MONAI_affine_Elastix_nonlinear.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.            ScaleIntensityRangeD(keys=["fixed_hand", "moving_hand"],
</pre><p>Dummy beginner question: how do these XxxD functions "know" that they are applied to the variable <code>training_datadict</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/ITKElastix/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>At this point it doesn't. However, in the next cell we do:</p><p> </p><pre class="ql-syntax" spellcheck="false">dataset = CacheDataset(data=training_datadict, transform=train_transforms)
dataloader = DataLoader(dataset=dataset, batch_size=16, shuffle=True)
</pre><p>where we supply the mentioned variable.</p>